### ISSUE TITLE
Console error when clicking on save button on media edit screen

### ISSUE THREAD

**Issue by SID177** [rtCamp/rtMedia#1662]
Below error is generated when clicking on **Save** button from media edit screen.
![84735227-94029400-afc0-11ea-8761-ea39578cf0f7](https://user-images.githubusercontent.com/23107546/85547609-b0937180-b63b-11ea-9994-3904d3297ee0.png)

**Reason**: This **Save** button has `imageEdit.save` JavaScript function call on it's click action, which is causing the error. This function call is only to save image which has been modified by an image editor, but it's getting called even when editing other media details as well, like Title/Description/.., which should be avoided.

**Comment by pooja-muchandikar** [comment]
Tested this issue with `pre-release` branch and added fix works as expected. 

```
Test Environment
rtMedia: v4.6.3 pre-release branch
rtMedia Set Custom Thumbnail add-on: v1.3.1 stable version
Theme: 2020 and 2019
```

**Comment by pooja-muchandikar** [comment]
Tested with `develop` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Released with`v4.6.4`
